### ISSUE TITLE
common/15 mixed pch is broken on Visual Studio 2015 (msvc 19)

### ISSUE THREAD

**Issue by nirbheek** [mesonbuild/meson#807]
```
ninja: Entering directory `build'
[1/5] Precompiling header ../pch/func_pch.c
FAILED: prog@exe/func.pch
cl  "/nologo" "/showIncludes" "/W2" "/MDd" "/ZI" "/Ob0" "/Od" "/RTC1" "/Ycfunc.h"  /Fpprog@exe\\func.pch" "/Fo prog@exe\\func.obj" "/FdC:\\Users\\nirbheek\\projects\\meson.git\\test cases\\common\\15 mixed pch\\build\\prog.pdb"   /c ../pch/func_pch.c
c:\users\nirbheek\projects\meson.git\test cases\common\15 mixed pch\pch\func_pch.c: fatal error C1041: cannot open program database 'C:\Users\nirbheek\projects\meson.git\test cases\common\15 mixed pch\build\prog.pdb'; if multiple CL.EXE write to the same .PDB file, please use /FS
[2/5] Precompiling header ../pch/main_pch.cc
ninja: build stopped: subcommand failed.
```

**Comment by nirbheek** [comment]
I've confirmed that it's working fine with Visual Studio 2010 (msvc 16).

**Comment by jpakkane** [comment]
Sadly that is all I have so I can't really debug this...

**Comment by nirbheek** [comment]
This was fixed in #995 via 88f1d400c070fdd95d00298584c6ac18cf087a14.
